databricks/sdk/core.py

```diff
@@ -614,6 +614,10 @@ def user_agent(self):
             ua.append(' '.join(self._user_agent_other_info))
         if len(self._upstream_user_agent) > 0:
             ua.append(self._upstream_user_agent)
+        if 'DATABRICKS_RUNTIME_VERSION' in os.environ:
+            runtime_version = os.environ['DATABRICKS_RUNTIME_VERSION']
+            if runtime_version != '':
+                ua.append(f'runtime/{runtime_version}')
```

Comment:
Please add a check that this value conforms to an expected pattern, e.g. no spaces/slashes.

Can be a simple regex match. 

I suspect you expect to see `13.3.x-scala2.12`?

Reply:
@pietern it's `13.3`